lib/frederick_api/v2/automation_step.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module FrederickAPI
+  module V2
+    # Resource for automation step
+    class AutomationStep < Resource
+      belongs_to :location
+      has_one :automation
```

Comment:
what makes it a `has_one` rather than `belongs_to`?

Reply:
I noticed that all `belongs_to` relationships from frobot and frolodex use `has_one` in this gem, except for location_id - I wonder if it has something to do with access token scope because when I changed automation to a `belongs_to`, I couldn't create automation steps from plat

Reply:
I saw that too.  Still wondering what that's about, but good to know you tested it :)